### ISSUE TITLE
getopt-generics-0.10: examples testsuite failing

### ISSUE THREAD

**Issue by juhp** [zalora/getopt-generics#47]
Noticed while trying to build stackage nightly:

Test run getopt-generics-0.10 (pending: 601, failures: 0)
getopt-generics: Process exited with ExitFailure 1

```
Test suite examples: RUNNING...
examples: docs: getDirectoryContents: does not exist (No such file or directory)
Test suite examples: FAIL
```

**Comment by soenkehahn** [comment]
Thanks for reporting. This should be fixed in `0.10.0.1`, released to hackage just now.

**Comment by juhp** [comment]
Thanks
